examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -30,144 +25,233 @@ func Render(_ string) string {
 		out, _ := events.RenderEventWidget(events.MaxWidgetSize)
 		return out
 	})
-	r.RegisterCallback(":latest-hof:", func() string {
-		return hof.RenderExhibWidget(5)
-	})
-	r.RegisterCallback(":qotb:", quoteOfTheBlock)
-	r.RegisterCallback(":chain-height:", func() string {
-		return strconv.Itoa(int(std.ChainHeight()))
-	})
 
 	template := `# Welcome to gno.land
 
-We’re building gno.land, set to become the leading open-source smart contract
-platform, using Gno, an interpreted and fully deterministic variation of the
-Go programming language for succinct and composable smart contracts.
+## We're building a user-owned internet where governance is fair, development is open, and innovation is rewarded.
```

Comment:
we should not use H2 just to make text bigger.

Reply:
I think it's a good idea to use it as an H2 for SEO purposes (not for text size).